config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -454,8 +454,8 @@ presubmits:
           value: '{"EventedPLEG":true}'
         - name: RUNTIME_CONFIG
           value: '{"api/alpha":"true", "api/ga":"true"}'
-        - name: FOCUS
```

Comment:
@pacoxu should the job be moved into `sig-node`?

Reply:
+1

Reply:
this can be done later 😄